dj/models/node.py

```diff
@@ -152,6 +182,15 @@ class Node(NodeBase, table=True):  # type: ignore
         },
     )
 
+    missing_parents: List[MissingParent] = Relationship(
+        link_model=NodeMissingParents,
+        sa_relationship_kwargs={
+            "primaryjoin": "Node.id==NodeMissingParents.referencing_node_id",
+            "secondaryjoin": "MissingParent.id==NodeMissingParents.missing_parent_id",
+            "cascade": "all, delete",
```

Comment:
I am not fully sure how SQL Alchemy resolves this but I think you DO want the cascading delete on the primary join, but I hope the cascading delete will not happen on the secondaryjoin. @betodealmeida any experience with this?

Reply:
I spent a little time reading up on how this works and I think this would be ok as currently defined because it's actually going to create a dedicated join table. So what's being cascaded are just the join records. If a missing parent reference is directly deleted (which I don't think will ever happen), then all of those links will be removed from the join table but no nodes are deleted. More likely is that a node is deleted in which case all of the links from that node to all missing parents will be removed from the join table.